op-chain-ops/genesis/layer_one.go

```diff
@@ -67,11 +68,12 @@ func BuildL1DeveloperGenesis(config *DeployConfig, dump *gstate.Dump, l1Deployme
 			memDB.CreateAccount(address)
 			memDB.SetNonce(address, account.Nonce)
 
-			balance, ok := new(big.Int).SetString(account.Balance, 10)
-			if !ok {
-				return nil, fmt.Errorf("failed to parse balance for %s", address)
+			var balance math.HexOrDecimal256
+			err := balance.UnmarshalText([]byte(account.Balance))
```

Comment:
Technically a cleaner implementation would use this: https://github.com/ethereum/go-ethereum/blob/cd0770ea6855a7704059aa7c591d0e83dcb21231/common/math/big.go#L117